### DISCUSSION TITLE
Permissions Database auth.ini

### DISCUSSION THREAD

**Discussion by javier5109** [mlflow/mlflow#9508] [Q&A] [answered]
I am have a question regarding the database auth.ini (mlflow/server/auth/basic_auth.ini)

I see that the basi_auth.db is stored locally. Is it possible to make use an external database? Or to use the same backend store uri for it?

**Comment by j-arpit** [comment]
You can. Clone the mlflow repository, Chage database uri in this file mlflow/server/auth/basic_auth.ini and build the package.

**Comment by B-Step62** [comment, marked as answer]
@javier5109 Thank you for opening this question!

Indeed MLflow support that (without cloning repo). You can set `MLFLOW_AUTH_CONFIG_PATH` environment variable to your custom `basic_auth.ini` file that defines configurations such as external database URL. (Ref: [code](https://github.com/mlflow/mlflow/blame/master/mlflow/server/auth/config.py#L18C9-L18C32)).

We are updating the auth doc to include this information (#10590).
